stock_inventory_include_exhausted/models/stock_inventory.py

```diff
@@ -18,6 +18,8 @@ def _get_inventory_lines_values(self):
 
         if self.include_exhausted:
             domain = [("qty_available", "=", 0), ("type", "=", "product")]
+            product_ids = [data.get("product_id") for data in vals]
```

Comment:
@nicomacr Thansk for this. Could you add a test that highlight this ?

Reply:
Ready!!